resources/user-agents/browsers/chrome/chrome-generic.json

```diff
@@ -48,7 +48,9 @@
           "match": "Mozilla/5.0 (#PLATFORM#)*applewebkit*(*khtml*like*gecko*)*Chrome/*",
           "device": "Macintosh",
           "platforms": [
-            "OSX_10_15", "OSX_10_14", "OSX_10_13", "OSX_10_12",
+            "OSX_12_0",
+            "OSX_11_0", "OSX_11_1",
```

Comment:
Please change the ordering. Please write `"OSX_11_1", "OSX_11_0"` (like you did in the Chrome file). Please change it in other files too.